x/nodes/types/params.go

```diff
@@ -11,65 +11,77 @@ import (
+	DefaultServicerStakeFloorMultiplier int64 = 15000000000
+	DefaultServicerStakeWeightCeiling   int64 = 15000000000
 )
 
 //  - Keys for parameter access
 var (
-	KeyUnstakingTime               = []byte("UnstakingTime")
-	KeyMaxValidators               = []byte("MaxValidators")
-	KeyStakeDenom                  = []byte("StakeDenom")
-	KeyStakeMinimum                = []byte("StakeMinimum")
-	KeyMaxEvidenceAge              = []byte("MaxEvidenceAge")
-	KeySignedBlocksWindow          = []byte("SignedBlocksWindow")
-	KeyMinSignedPerWindow          = []byte("MinSignedPerWindow")
-	KeyDowntimeJailDuration        = []byte("DowntimeJailDuration")
-	KeySlashFractionDoubleSign     = []byte("SlashFractionDoubleSign")
-	KeySlashFractionDowntime       = []byte("SlashFractionDowntime")
-	KeyRelaysToTokensMultiplier    = []byte("RelaysToTokensMultiplier")
-	KeySessionBlock                = []byte("BlocksPerSession")
-	KeyDAOAllocation               = []byte("DAOAllocation")
-	KeyProposerAllocation          = []byte("ProposerPercentage")
-	KeyMaxChains                   = []byte("MaximumChains")
-	KeyMaxJailedBlocks             = []byte("MaxJailedBlocks")
-	DoubleSignJailEndTime          = time.Unix(253402300799, 0) // forever
-	DefaultMinSignedPerWindow      = sdk.NewDecWithPrec(5, 1)
-	DefaultSlashFractionDoubleSign = sdk.NewDec(1).Quo(sdk.NewDec(20))
-	DefaultSlashFractionDowntime   = sdk.NewDec(1).Quo(sdk.NewDec(100))
+	KeyUnstakingTime                            = []byte("UnstakingTime")
+	KeyMaxValidators                            = []byte("MaxValidators")
+	KeyStakeDenom                               = []byte("StakeDenom")
+	KeyStakeMinimum                             = []byte("StakeMinimum")
+	KeyMaxEvidenceAge                           = []byte("MaxEvidenceAge")
+	KeySignedBlocksWindow                       = []byte("SignedBlocksWindow")
+	KeyMinSignedPerWindow                       = []byte("MinSignedPerWindow")
+	KeyDowntimeJailDuration                     = []byte("DowntimeJailDuration")
+	KeySlashFractionDoubleSign                  = []byte("SlashFractionDoubleSign")
+	KeySlashFractionDowntime                    = []byte("SlashFractionDowntime")
+	KeyRelaysToTokensMultiplier                 = []byte("RelaysToTokensMultiplier")
+	KeySessionBlock                             = []byte("BlocksPerSession")
+	KeyDAOAllocation                            = []byte("DAOAllocation")
+	KeyProposerAllocation                       = []byte("ProposerPercentage")
+	KeyMaxChains                                = []byte("MaximumChains")
+	KeyMaxJailedBlocks                          = []byte("MaxJailedBlocks")
+	KeyServicerStakeFloorMultiplier             = []byte("ServicerStakeFloorMultiplier")
+	KeyServicerStakeWeightMultiplier            = []byte("ServicerStakeWeightMultiplier")
+	KeyServicerStakeWeightCeiling               = []byte("ServicerStakeWeightCeiling")
+	KeyServicerStakeFloorMultiplierExponent     = []byte("ServicerStakeFloorMultiplerExponent")
```

Comment:
typo should be : []byte("ServicerStakeFloorMultiplierExponent"

Reply:
Good spot! fixed in the squashed commit